CONTRIBUTING.md

```diff
@@ -1,18 +1,29 @@
-Sonata respects the Symfony's conventions about contributing to the code. So before going further please review the [contributing documentation of Symfony](http://symfony.com/doc/current/contributing/code/patches.html#make-a-pull-request).
+Sonata respects the Symfony's conventions about contributing to the code. So
+before going further please review the [contributing documentation of
+Symfony](http://symfony.com/doc/current/contributing/code/patches.html#make-a-pull-request).
 
 ## Reporting bugs
 
-If you happen to find a bug, we kindly request you to report it. However, before submitting it, please:
+If you are not sure this is a bug, consider posting your question on [Stack
+Overflow](http://stackoverflow.com), using one of the sonata tags.
+If you happen to find a bug, we kindly request you to report it. However,
+before submitting it, please check the [project documentation available
+online](https://sonata-project.org/bundles/)
 
-  * Check the [project documentation available online](https://sonata-project.org/bundles/)
+Then, if it appears that it's a real bug, you may report it using Github by
+following these 4 points:
 
-Then, if it appears that it's a real bug, you may report it using Github by following these 3 points:
+* Check if the bug is not already reported!
+* A clear title to sum up the issue
+* A description of the workflow needed to reproduce the bug
+* If your issue is an error page, you must provide us with a stack trace.  With
+  recent version of Symfony, you can even get stack traces as plain text at the
+end of the page. Just look for "Stack Trace (Plain Text)", and copy/paste what
+you see. **Do not** make a screenshot, as screenshots are not indexed by search
+engines and will make it difficult for other people to find your bug report
 
```

Comment:
I meant "don't provide" a screenshot of the stack trace, but adding what you said may help to clarify. Won't be able to do that until Tuesday though.